users/danielo515/process_records.c

```diff
@@ -97,6 +95,11 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
                 layer_off(_ADJUST);
             }
             return false;
+        case GAMING:
+            if (record->event.pressed) {
+                layer_on(_GAMING);
```

Comment:
This could be replaced with `TG(_GAMING)` in the keymap. 

otherwise, `layer_invert(_GAMING)` may be more appropriate here (as that toggles it).

Reply:
The reason why I went this route was... not sure why because now that I think about it I'm not directly calling the function that updates the leds here, so I don't know what I was thinking about. I'll change it.